pkg/controller/persistentvolume/persistentvolume_provisioner_controller.go

```diff
@@ -506,6 +506,10 @@ func (c *PersistentVolumeProvisionerController) NewWrapperMounter(volName string
 	return nil, fmt.Errorf("NewWrapperMounter not supported by PVClaimBinder's VolumeHost implementation")
 }
 
+func (c *PersistentVolumeProvisionerController) NewWrapperAttacher(volName string, spec volume.Spec, pod *api.Pod) (volume.Attacher, error) {
```

Comment:
Looks like the wrapper is from https://github.com/kubernetes/kubernetes/pull/25502. We can address comments on it there.